mesonbuild/environment.py

```diff
@@ -17,6 +17,7 @@
 import re
 
 from .compilers import *
+from .linkers import *
```

Comment:
__[flake8]__
* [F403] 'from .linkers import *' used; unable to detect undefined names

<!-- Comment by [SideCI](https://sideci.com) -->